docs/known_limitations.md

```diff
@@ -0,0 +1,21 @@
+# Known Limitations
+
+## Purpose
+
+This set of documentation describes the next generation Citrine data format, code-named GEMD, that is independent of any implementation of the format.  
```

Comment:
Isn't taurus the codename and GEMD the real name?

Reply:
Yes, and I don't know how my brain turned off there.